docs/current_docs/tutorials/934191-eks-github-karpenter.mdx

```diff
@@ -0,0 +1,108 @@
+
+## Step 3: Create a set of taints and tolerations for the GitHub Actions runners
+
+[Taints and tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) in Kubernetes are a way to ensure that certain nodes are reserved for specific tasks. By setting up taints on runner nodes, you can prevent other workloads from being scheduled on them. Tolerations are then added to the runners so that they can be scheduled on these tainted nodes. This ensures that the runners have dedicated resources for their tasks.
+
+A sample GitHub Actions runner deployment configuration is shown below.
+
+- Replace the `YOUR_GITHUB_ORGANIZATION` placeholder with your GitHub organization name. If you do not have a GitHub organization, you can use your GitHub username instead.
+- This configuration also uses the `DAGGER_CLOUD_ENVIRONMENT` environment variable to connect this Dagger Engine to Dagger Cloud. Replace `YOUR_DAGGER_CLOUD_TOKEN` with your own Dagger Cloud token.
+
+```yaml file=./snippets/eks-github-karpenter/runner_deployment.yml
+```
+
+:::note
+This configuration uses the `_EXPERIMENTAL_DAGGER_RUNNER_HOST` environment variable to point to the Dagger Engine DaemonSet socket that is mounted into the GitHub Actions runners. This ensures that the runners will use the local Dagger Engine for pipeline execution.
+:::
+
+## Step 4: Deploy the Dagger Engine DaemonSet with Helm v3
+
+A Dagger Engine is required on each of the GitHub Actions runner nodes. A DaemonSet ensures that all matching nodes run an instance of Dagger Engine. To ensure that the Dagger Engines are co-located with the GitHub Actions runners, the Dagger Engine Daemonset should be configured with the same taints and tolerations as the GitHub Actions runners.
+
+Use our Helm chart to create the Dagger Engine DaemonSet on the cluster:
+
+```shell
+helm upgrade --install --namespace=dagger --create-namespace \
+    dagger oci://registry.dagger.io/dagger-helm
+```
+
+This Dagger Engine DaemonSet configuration is designed to:
+
+- best utilize local Non-Volatile Memory Express (NVMe) hard drives of the worker nodes
+- reduce the amount of network latency and bandwidth requirements
+- simplify routing of Dagger SDK and CLI requests
+
+## Step 5: Test the deployment
+
+At this point, the deployment is configured and ready for use. Test it by triggering the GitHub Actions workflow, by committing a new change to the source code repository. Your CI pipelines will be now be connected to your Dagger Engines.
+
+If you don't already have a GitHub repository, clone the [Dagger open source repository](https://github.com/dagger/dagger) and add the sample GitHub actions workflow shown below to it. The workflow uses the Dagger for GitHub Action to execute a Dagger Function that builds a Go binary of the Dagger CLI. Refer to the inline comments for configuration you may wish to change.
```

Comment:
This section has changed from the previous version of the guide. Instead of using a sample Node app and a custom SDK-based script, I have updated it to use Kyle's Go builder module and the Dagger repo. Let me know if this is fine?